### PR TITLE
Add multiple command line email address handling for find, add, remove

### PR DESCRIPTION
**Pull request by jaleksynas**
-- E.g.  pilot find foo@bar.com bar@baz.org <options>
-- E.g.  pilot add foo@bar.com bar@baz.org <options>
-- E.g.  pilot remove foo@bar.com bar@baz.org <options>
-- If no addresses are specified, you will be prompted for one.
-- If any exception occurs, log it and proceed to next element
-- after all are processed, raise exception if any of them failed.

### CODE REVIEW

File: lib/pilot/commands_generator.rb [hunk 1]
```diff
@@ -25,9 +25,21 @@ def convert_options(options)
       o
     end
 
-    def handle_email(config, args)
-      config[:email] ||= args.first
-      config[:email] ||= ask("Email address of the tester: ".yellow)
+    def handle_multiple(action, args, options)
+      mgr = Pilot::TesterManager.new
+      config = FastlaneCore::Configuration.create(Pilot::Options.available_options, convert_options(options))
+      args.push( ask("Email address of the tester: ".yellow) ) if args.empty?
```
Comment (KrauseFx) [inline lib/pilot/commands_generator.rb]: Please remove the space after the `(`

**Comment by KrauseFx** [comment]
Looks good, I added one comment. Also would like a confirmation from @samrobbins :+1: 

I don't know if that's caused by this PR, but could you fix that one styling issue: https://travis-ci.org/fastlane/pilot/jobs/104738919?

**Comment by tmm1** [comment]
:+1: would love to see this merged

**Comment by tmm1** [comment]
Confirmed this works as expected for me.

**Comment by KrauseFx** [comment]
Perfect, thanks for confirming @tmm1, working on a new release :rocket:

**Pull request merged**